### ISSUE TITLE
YouTube RegexNotFoundError

### ISSUE THREAD

**Issue by breadcat** [rg3/youtube-dl#4711]
I'm running:
```
youtube-dl -x --audio-format=mp3 https://www.youtube.com/watch?v=Xp4WKejt12Y --verbose
```

I'm receiving:
```
[debug] System config: []
[debug] User config: []
[debug] Command-line args: ['-x', '--audio-format=mp3', 'https://www.youtube.com/watch?v=Xp4WKejt12Y', '--verbose']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2015.01.15
[debug] Python version 2.7.3 - Linux-3.2.0-4-amd64-x86_64-with-debian-7.8
[debug] exe versions: avconv 0.8.16-6, avprobe 0.8.16-6, ffmpeg 0.8.16-6, ffprobe 0.8.16-6, rtmpdump 2.4
[debug] Proxy map: {}
[youtube] Xp4WKejt12Y: Downloading webpage
[youtube] Xp4WKejt12Y: Extracting video information
[youtube] {22} signature length 45.40, html5 player en_US-vfl86Quee
[youtube] Xp4WKejt12Y: Downloading js player en_US-vfl86Quee
ERROR: Signature extraction failed: Traceback (most recent call last):
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/youtube.py", line 617, in _decrypt_signature
    video_id, player_url, s
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/youtube.py", line 531, in _extract_signature_function
    res = self._parse_sig_js(code)
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/youtube.py", line 592, in _parse_sig_js
    'Initial JS player signature function name')
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/common.py", line 516, in _search_regex
    raise RegexNotFoundError('Unable to extract %s' % _name)
RegexNotFoundError: Unable to extract Initial JS player signature function name; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
 (caused by RegexNotFoundError(u'Unable to extract \x1b[0;34mInitial JS player signature function name\x1b[0m; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.',)); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/youtube.py", line 617, in _decrypt_signature
    video_id, player_url, s
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/youtube.py", line 531, in _extract_signature_function
    res = self._parse_sig_js(code)
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/youtube.py", line 592, in _parse_sig_js
    'Initial JS player signature function name')
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/common.py", line 516, in _search_regex
    raise RegexNotFoundError('Unable to extract %s' % _name)
RegexNotFoundError: Unable to extract Initial JS player signature function name; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 600, in extract_info
    ie_result = ie.extract(url)
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/common.py", line 260, in extract
    return self._real_extract(url)
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/youtube.py", line 1035, in _real_extract
    encrypted_sig, video_id, player_url, age_gate)
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/youtube.py", line 627, in _decrypt_signature
    'Signature extraction failed: ' + tb, cause=e)
ExtractorError: Signature extraction failed: Traceback (most recent call last):
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/youtube.py", line 617, in _decrypt_signature
    video_id, player_url, s
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/youtube.py", line 531, in _extract_signature_function
    res = self._parse_sig_js(code)
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/youtube.py", line 592, in _parse_sig_js
    'Initial JS player signature function name')
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/common.py", line 516, in _search_regex
    raise RegexNotFoundError('Unable to extract %s' % _name)
RegexNotFoundError: Unable to extract Initial JS player signature function name; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
 (caused by RegexNotFoundError(u'Unable to extract \x1b[0;34mInitial JS player signature function name\x1b[0m; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.',)); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
```

This is the only video I've found an issue with.
Running on Debian stable, everything up to date, latest youtube-dl.
I've been able to grab an m4a using Jdownloader2.

**Comment by psycotica0** [comment]
I also thought it was too, but it turns out that was fixed in master 2 hours ago.
Your url works for me, having just updated.

**Comment by psycotica0** [comment]
Sorry, bad form. It seems to work as of https://github.com/rg3/youtube-dl/commit/a9b6b5cd15cff004b4109ab64b181fff23c10770

**Comment by dstftw** [comment]
This is already fixed and will be available in the next release. Thanks for the report.

**Comment by dstftw** [comment]
This issue was fixed in youtube-dl 2015.01.15.1. Refer to [our FAQ](https://yt-dl.org/update) if you need help updating.
